mixer/cmd/build.go

```diff
@@ -110,6 +112,144 @@ var buildBundlesCmd = &cobra.Command{
 	},
 }
 
+// buildOldFormatCmd is used to build the final version in the current format (the +10)
+// and ready the mix state for the new (+20) version to be built by a (possibly) newer mixer
+var buildOldFormatCmd = &cobra.Command{
+	Use:   "format-old",
```

Comment:
The `mixer build format-old` and `mixer build format-new` names aren't particularly intuitive to me. Maybe "last", like you use in the function descriptions?

Reply:
I had a long discussion with myself about how to name these and then gave up and left it as is to finish the rest of the code. I am :100:  looking for suggestions on how to name these better.